deployer/templates/heapster.yaml

```diff
@@ -84,9 +86,9 @@ objects:
           - "--wrapper.endpoint_check=https://hawkular-metrics:443/hawkular/metrics/status"
           - "--source=kubernetes.summary_api:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250"
           - "--sink=hawkular:https://hawkular-metrics:443?tenant=_system&labelToTenant=pod_namespace&labelNodeId=${NODE_ID}&caCert=/hawkular-cert/hawkular-metrics-ca.certificate&user=%username%&pass=%password%&filter=label(container_name:^system.slice.*|^user.slice)"
-          - "--tls_cert=/secrets/heapster.cert"
-          - "--tls_key=/secrets/heapster.key"
-          - "--tls_client_ca=/secrets/heapster.client-ca"
+          - "--tls_cert=/heapster-certs/tls.crt"
+          - "--tls_key=/heapster-certs/tls.key"
+          - "--tls_client_ca=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
```

Comment:
This is the wrong ca, its needs to the main ca, not the service-ca.crt
eg: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"